hw/ip/tlul/doc/dv_plan/index.md

```diff
@@ -108,8 +108,7 @@ We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/RE
 Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
 Here's how to run a basic sanity test:
 ```console
-$ cd hw/ip/xbar/dv
-$ make TEST_NAME=xbar_sanity
+$ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_main_sim_cfg.hjson -i xbar_main_sanity
```

Comment:
how about this?
```suggestion
$ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/top_earlgrey/ip/$XBAR_IP/dv/autogen/xbar_main_sim_cfg.hjson -i ${XBAR_IP}_sanity
// Note: $XBAR_IP could be xbar_main, xbar_peri, etc.
```

Reply:
that works thanks Weicai for the suggestion. Let us discuss more with Sri if we should talk about anything top-level here. If so, will definitely use your suggestion.

Reply:
Also replace `top_earlgrey` with $CHIP as well and add a note indicating $CHIP = top_earlgrey, etc. This LGTM.